src/draw/vg_lite/lv_vg_lite_utils.c

```diff
@@ -328,7 +328,7 @@ bool lv_vg_lite_is_src_cf_supported(lv_color_format_t cf)
 {
     switch(cf) {
         case LV_COLOR_FORMAT_A4:
-        case LV_COLOR_FORMAT_A8:
+        // case LV_COLOR_FORMAT_A8:
```

Comment:
Accidentally left here commented (because of debugging), or it is really not needed?
In latter case you could remove it, git history will have it anyway.